.github/workflows/stable-spec-tests.yml

```diff
@@ -0,0 +1,82 @@
+name: Execution Spec Tests - Consume (stable)
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master, kaustinen-with-shapella]
+  workflow_dispatch:
+
+env:
+  FIXTURES_TAG: "verkle@v0.0.3"
```

Comment:
We should use [the corresponding tag in the release](https://github.com/ethereum/execution-spec-tests/releases/tag/verkle%40v0.0.3) we want to target.